witness/golang/internal/witness/errors.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package witness
+
+import "fmt"
+
+// ErrInconsistency should be returned when there has been an error proving
+// consistency between two checkpoints.
+type ErrInconsistency struct {
```

Comment:
This type is unused, except in the place where it's returned. I would maybe postpone introducing this type until it's clear how it's going to be used. Right now, in the place where it's returned, it could just as well return `fmt.Errrorf(...)`.

Reply:
Sure, I agree.  Changed this and removed `errors.go` as a result.